file-formats/cfdf/type/zif_aff_cfdf_v1.intf.abap

```diff
@@ -0,0 +1,589 @@
+INTERFACE zif_aff_cfdf_v1
+  PUBLIC.
+
+  TYPES: ty_field_name              TYPE c LENGTH 30.
+  TYPES: ty_component_identifier    TYPE c LENGTH 5.
+  TYPES: ty_business_context        TYPE c LENGTH 30.
+  TYPES: ty_code_value              TYPE c LENGTH 30.
+  TYPES: ty_data_subject_id_type    TYPE c LENGTH 10.
+  TYPES: ty_type                    TYPE c LENGTH 10.
+  TYPES: ty_message_direction       TYPE c LENGTH 3.
+  TYPES: ty_field_control_property  TYPE c LENGTH 1.
+  TYPES: ty_aggregation             TYPE c LENGTH 5.
+
+  TYPES:
+    "! <p class="shorttext">Label and Tooltip</p>
+    "! Label and tooltip
+    BEGIN OF ty_gs_ui_text,
+      "! <p class="shorttext">Translation Language</p>
+      "! Translation language
+      "! $required
+      language      TYPE sy-langu,
+
+      "! <p class="shorttext">Field Label</p>
+      "! Field label
+      "! $required
+      "! $maxLength 40
```

Comment:
If you use `TYPE c LENTGH 40`, it is not necessary to specify maxLength 40 again. It will be added to the JSON schema even without the annotation since it is derived from the type. Same applies to the other fields with $maxLength annotation

Reply:
All occurrences of $maxLength were removed...